org.civicrm.bootstrapcivihr/scss/pages/loader.scss

```diff
@@ -0,0 +1,10 @@
herSF5MBNgJM4Aqk+yiqYrTo8HOABhUDGGDgMGxhuB8RTqz4ugCHWAW+YIWzFIAId4UMpP05YaGox+6WNKVraS8breuBLjjlqC7WmhSUDBwNIWaX93OGdRm60OAUAXhaeq5M0IRQbCuwUuoTVUAD8EKqpMRgECNOJl8BWvGbqG3OQGGoCv4jhIBCZOC9ToGDmqOAGoOhpMTwMCecXUXajOgoJTBc79qxQmAwJLGWgEuVDNYjlYuYXsCLIA0EU6AGwi7wnn85UZ7p4YxAsBJ2M5Zr19rAig4tbCaUVQLyR37lUY+/YSbGMAANAzB440xZ8jHSUJcBNBDRg25k80b8gkyWUHr2BAAHtSY5E4hzM2RzYUC+ZJBAKhEM5M8jjB3BzDhDOSaCLwQuGKLBvhRizHouCNGAF/sYuOKjRiQ4y668KKhKbkMORM8jRDg2zGLGCDBME2uSFAj8hUSSQA+2JKllgO1iEAHAaBRy5hkCtRib75Qw0w7bRL0iANMaFPnQcvEqFBAACH5BAEFAP8ALAAAAAAgACAAAAj+AOUJHEiwoMGDCBMqXMiwocOHECOaOxcxoTtvzXzp0jWu4sB357Id04VLFy+S4gyiG5arpcuXuGSJkqRI0SJGjCyBetQolq5yBFEFGEq0qNGjRvEQ5DSUDRsvWKJaqXJFDBw7d7JqtUMGi5YHAb4Q9HYjwJJgtWjNojWM2jh069i1m0uX3S9fbggESCnQWzA7BgJMGgYt3DuG7nKNQhFg0UB4xW4BKwsDYrdaYwKEYEdQmy5itBAQiPSQGScHAVgZ9HwsS4AV2hz+IhLACdDVu2JJCMCnYbpGASbUSui5T4AOxxh2sxAgy0JtsmgEUKNuoZwAF6AplzR0mEJvYN+pOKTWJAARdwnDBMCA3uEt5qEQ8poQwFLEN8GrqXRtw1vEcBsEkARFBNEylCftVGRJAAhswpc84wgRwBbBeCRPCwHM8Es4An0SQAWcWGPhMQzyEUw44VAQwBm3EOgRGwG8cIoyYASQAivBtOeRNh0EcEgkBBiACC/NwGOhQJkEcIIHAbQwjC2xHSlPN0AQZQkvu5gjpUCeBHbGOb0wk+CW7CBSBkXo6IhQQAAh+QQBBQD/ACwAAAAAIAAgAAAI/gDlCRxIsKDBgwgTKlzIsKHDhw2t1YLYMBoDAguOWAJHESEtAgEMBBjpIQ6wceneUXyHThwCA3agmKAwMgCCI490NdN2rmc5cuV+lnNHEFwuYDECwDkWa9KaIikQ1NTAI0wgTrl64aJFq1Y1gt1w9SIUQAg4a9m0VRPmSlKaFTURgNiBplOzY7+iEXR3DJevACvCHVTnDVqmKFIDEHCgwYebawW98eq1QcJEhfDkBZNzoqZiIuoIKts1JcAhiMrm+ABJ4RfBc70aBchCtGE6Yb0WSfVSeyA1VgRe6G0IrRYbkGlCF3yXqwYBXA3L5Yojks86hN7KBIDUkJickXqUwR+ENyqAlXMLtc0ZuSbdQmgdIljDvGdkmHYMz2kJUEqhopFfPPSfF96MB4lIZfTG0CwOnPCLSgVJIpUa7j10DQwBzAJNQZuMtMU4HUURACDAoDOQKSNFMRxFiAQwxC/KCJTJSEXscl1HwDzAwC+/eCMJSEfQ0k1HA4UQACu4zCEVFbQcoyBFVwSQByEgqUFLLwUSKRAoAUgg1Ri65HKMlgOJA1IATcySiy7okSlQEhV8kU0vt6zo5jvDEJVNNBAyFBAAIfkEAQUA/wAsAAAAACAAIAAACP4A5QkcSLCgwYMIEypcWBAdqk3gGEocOCxAgBydJi5cVy6XxQAKchDTeDBdr12OApC5YtHAmHMkCZLTpatRgCu8RI0gEEBDqXQxBfbqVSpAF16+fM3xYPHILXQxlfEaFYBOOG3RnLHaUsGil1jl2k3slitSgDvuBLJD940WGIsOrMiilq3cQne4GJ0V23DZEbhoegULxqwbX4PEFu1NqCqDxQ2PfuXKpUsZPIPWFC1O6E5OA4tQXlH+Zc6guUSbFTqL4iCAhTajqh189yj1wlQ+LJawBNPgJ9sKwcHyMyMAgUAHXwFH2G2XsE0ULKY56PHNOobecuG2SCDOO+oBwKlAVehNFZwFARAEyZbQY5VxCqn9WWExR6qF7qkldAXE4gpH4jDkkRXCHKSNF60ZMIg1E3mExSzfEeRHawFIEU1aDQaAhSzhDLQLCBZlMEpQHm1BCzTyMLOFRRF4wUxQ8ni0xi+iHIJeA0J0og2MMQZgBhofWAREIrlYcxmMHj3A0wmawFLkkTDq8lEauexyi5E8ClQOHl94w8tk0UCZ5UDL/ILimAf1FlNAACH5BAEFAP8ALAAAAAAgACAAAAj+AOUJHEhQ3jt17woqXMhwYCMalhpKVPgOnLd0AQIYgARvosdzuXxBCtBAYyKPE8nVCrYkABU7GRWhbFiOlrYQARrtshJz5sJyt1pJIMBsly4mBAJM8lmwHC86BJLI61aMV5GMizoyleeUSABNAttlA/ZEo8yt5k6hCOCNoDpoXjQCcsf0XKIGJ7QSFPcmIxpwPtOtCSCHobu+AcwcM4fy25EAtiSOSTpGV7R1E6GBQKBtYpqMYGgB04aZYbAASch51BOAgBlbtop1S1gQnqYAgehOhCeo9Zxjt3RdU7gOS4BRM9kBylgm2K1oCstRuIDL57kwoLMtXBZARzOU7rrEBevlJKmhhYECaOm2e1yzWqMoGVoQYMEwhTEM/Onc0N0wSWo0gQICGQXABHsEnRPABqQMt9A1jjAxwgQFSvCEJbzwR5ApAdQAjDV6yaMLHSY4YECBJ+SxyzjohEiQcXP0Qo074mgSBYEFMpBEJ+fMBAECmigShgkFGhABCmK0shUxGVFQoAYvPFHHKg5udUxGCLRwBiGYtBJMNIxtJRA8vowSTCm9/JILMeLQJiZB5tSCiy7auPimQO5UA41ud/bp550BAQA7) no-repeat center center !important;
```

Comment:
Does this mean that there are still two spinners on the page, is this expected?

Reply:
> Does this mean that there are still two spinners on the page, is this expected?

Yes, we have two spinners: one defined in civihr repos and another coming form code civicrm.
1. Currently, the spinners are in shoreditch, hrjobcontract, T&A and reqangular repo which are same and slowly we will keep the spinner in one place.
2. And there is another spinner in core civicrm which is different.

This PR simply overrides the core CiviCRM spinner to make is consistent.